app/src/ui/changes/commit-message.tsx

```diff
@@ -387,7 +391,7 @@ export class CommitMessage extends React.Component<
       action: this.onToggleCoAuthors,
       enabled:
         this.props.repository.gitHubRepository !== null &&
-        !this.props.isCommitting,
+        !(this.props.isCommitting === true),
```

Comment:
This should work too, right?
```suggestion
        this.props.isCommitting !== true,
```